lntest/itest/assertions.go

```diff
@@ -1116,6 +1116,63 @@ func assertAmountSent(amt btcutil.Amount, sndr, rcvr *lntest.HarnessNode) func()
 	}
 }
 
+// assertEdgeDisabled checks that a given edge for a channel in a node has the
+// Disable state. Passing a boolean for incoming allows distinction between the
+// outgoing and incoming edge for the channel specified by channel point
+func assertEdgeDisabled(t *harnessTest, node *lntest.HarnessNode,
```

Comment:
I think replacing the inline `assertEdgeDisabled` in `lnd_channel_graph_test.go` starting on line 109 could still be replaced by this to get some duplicated code removed.